templates/core/index.html

```diff
@@ -1,817 +1,753 @@
-  <div class="row u-vertically-center">
-    <div class="col-8">
-      <h2>Built for business</h2>
-      <p class="p-heading--4 u-no-margin--bottom">Bring your applications, we take care of the rest.</p>
-    </div>
-  </div>
-</section>
-
-<section class="p-strip">
-  <div class="row u-vertically-center u-equal-height">
-    <div class="col-6">
-      <h3 class="p-heading--4">Enterprise management</h3>
-      <p class="p-heading--5">Absolute control over every device in the building.</p>
-      <p>Enterprises gain complete control over every app on every device on the network, regardless of manufacturer. Know exactly which kernel version and which apps are running.</p>
-    </div>
-    <div class="col-6 u-hide--medium u-hide--small u-align--center u-vertically-center">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/e68fd355-1+discovery_AW.svg",
-        alt="",
-        width="350",
-        height="200",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
-    </div>
-  </div>
-
-  <div class="u-fixed-width">
-    <hr class="p-separator" />
-  </div>
-
-  <div class="row u-vertically-center">
-    <div class="col-6 u-hide--medium u-hide--small u-align--center">
-      {{
-        image(
-        url="https://assets.ubuntu.com/v1/4e20332d-mission+critical+support.svg",
-        alt="",
-        height="200",
-        width="350",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
-    </div>
-    <div class="col-6">
-      <h3 class="p-heading--4">Mission critical support</h3>
-      <p class="p-heading--5">Canonical supports Ubuntu Core devices</p>
-      <p>World class support - for product teams and enterprise customers. Get to the heart of a problem faster, get fixes straight from the source.</p>
-      <p><a href="/core/contact-us?product=core-overview" class="p-button--positive js-invoke-modal">Contact us</a></p>
+    <div class="col-3 col-medium-2">
+      <p>
+        <a href="/core/contact-us">Contact us&nbsp;&rsaquo;</a>
+      </p>
     </div>
   </div>
 </section>
 
-<section class="p-strip--light" id="developers-love-ubuntu">
-  <div class="row u-vertically-center">
-    <div class="col-8">
-      <h2>Developers love Ubuntu</h2>
-    </div>
-  </div>
-  <div class="row">
-    <div class="col-5">
-      <p class="p-heading--4">Your business is software defined.<br>Keep developers happy and productive.</p>
-    </div>
-    <div class="col-6 col-start-large-7">
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">Happy developers create amazing products</li>
-        <li class="p-list__item is-ticked">Amazing products attract more great developers</li>
-        <li class="p-list__item is-ticked">8/10 Linux developers choose Ubuntu</li>
-      </ul>
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>Learn more about Ubuntu Core</h2>
```

Comment:
```suggestion
      <h2>Learn more<br />about Ubuntu Core</h2>
```
Same as above, this or a `nbsp`, whichever makes most sense